src/stories/MessageBar.tsx

```diff
@@ -0,0 +1,33 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import MessageBar, { MessageBarType } from '../components/MessageBar';
+import FakeLink from '../components/FakeLink';
+import withReadme from 'storybook-readme/with-readme';
+const readme = require('../components/MessageBar/README.md');
+
+storiesOf('MessageBar', module)
+  .addDecorator(withReadme(readme))
+  .add('basic usage', () => (
+    <div>
+      <MessageBar ariaLabel="A sample notice informing you about something">
+        Sample notice, something you should know about.
+      </MessageBar>
+    </div>
+  ))
+  .add('with a type', () => (
+    <div>
+      <MessageBar ariaLabel="Error describing what went wrong" type={MessageBarType.WARNING}>
+        Watch out, something unexpected might happen.
+      </MessageBar>
+    </div>
+  ))
+  .add('with an action', () => {
```

Comment:
Could we have another story with an icon action, like in Zeplin? Just to check that the spacing is all correct.